source/getting-started.txt

```diff
@@ -0,0 +1,135 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Tutorials </tutorials>
+   Template Apps </template-apps>
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _app-services-get-started-overview:
+
+The resources on this page are designed to help you explore Atlas App 
+Services. To explore a working codebase that showcases App Services' 
+features, check out a Template App. For a more guided experience in
+learning how to develop with App Services and Realm, start with a 
+Tutorial.
+
+.. _app-services-get-started-template-apps:
+
+Template Apps
+-------------
+
+Bootstrap App Services with a :doc:`Template App </template-apps>`. Template apps bring together
+many of the building blocks available in Atlas App Services and start you 
+off with a prebuilt app that you can customize. The following template
+apps are available:
+
+- A mobile todo app written with Realm SDKs that syncs to an App Services App
+- A hosted todo web app using the GraphQL API or Realm Web SDK
+- An Event-driven :doc:`Database Trigger </triggers/database-triggers>` 
+  template that updates a view in a separate collection.
+
+Template apps are available for the following languages:
+
+- JavaScript (React Native)
+- Kotlin (Android)
+- Swift (iOS + SwiftUI)
+- ...and more!
+
```

Comment:
```suggestion
```

i think we can delete this b/c it's not complete list and this info is covered elsewhere